c2corg_ui/templates/image/edit.html

```diff
@@ -141,8 +141,8 @@ <h2 class="heading show-phone" translate>general informations</h2>
 
               <div ng-class="{ 'has-success': image.image_type}">
                 <label><span translate>image_type</span> *</label>
-                <div class="input-group">
-                  <select class="form-control" ng-model="image.image_type" ng-options="rat as mainCtrl.translate(rat) for rat in ${image_types}"><option value=""></option></select>
+                <div class="input-group" ng-init="image_types = ${image_types}">
+                  <select class="form-control" ng-model="image.image_type" ng-options="rat as mainCtrl.translate(rat) for rat in image_types"><option value=""></option></select>
```

Comment:
Minor but I would recommend to use ``type`` instead of ``rat`` here because ``rat`` is probably a copy-paste from another part of the code, related to "ratings". ``rat`` is not really consistent here.